docs/src/pages/components/data-grid/pagination/pagination.md

```diff
@@ -43,6 +43,8 @@ Finally, you need to handle the `onPageChange` callback to load the rows for the
 
 {{"demo": "pages/components/data-grid/pagination/ServerPaginationGrid.js", "bg": "inline"}}
 
+**Note**: For more information regarding server-side pagination in combination with controlled selection check [here](/components/data-grid/selection/#usage-with-server-side-pagination)
```

Comment:
Two feedback on this one. A sentence should have a closing dot. "here" is considered to be avoided as link description: https://www.w3.org/QA/Tips/noClickHere. Maybe?

```suggestion
**Note**: For more information regarding server-side pagination in combination with controlled selection check [the dedicated section](/components/data-grid/selection/#usage-with-server-side-pagination).
```